lib/apr/notifications.ex

```diff
@@ -3,69 +3,70 @@ defmodule Apr.Notifications do
   require Logger
 
   def receive_event(event, topic, routing_key) do
-    event
-    |> slack_message(topic, routing_key)
-    |> post_message(topic, routing_key)
+    get_subscriptions(topic, routing_key)
+    |> Enum.map(&{&1, slack_message(&1, event, topic, routing_key)})
+    |> Enum.map(&post_message(&1))
```

Comment:
i _think_ this can change to 
```elixir
|> Enum.map(&post_message/1)
```
by default first argument is passed to the passed function, that `/1` just defines the function arity.